package com.example.gifbottary.domain.order.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import com.example.gifbottary.domain.order.entity.Order;

public interface OrderRepository extends JpaRepository<Order, Long> {
}
